.github/workflows/xk6-tests/xk6-js-test/jstest.go

```diff
@@ -58,12 +58,11 @@ func (j *JSTest) Foo(arg float64) (bool, error) {
 
 	ctx := j.vu.Context()
 
-	tags := state.CloneTags()
-	tags["foo"] = "bar"
+	tags := state.Tags.GetCurrentValues().With("foo", "bar")
 	metrics.PushIfNotDone(ctx, state.Samples, metrics.Sample{
-		Time:   time.Now(),
-		Metric: j.foos, Tags: metrics.IntoSampleTags(&tags),
-		Value: arg,
+		Time:       time.Now(),
+		TimeSeries: {Metric: j.foos, Tags: tags},
```

Comment:
```suggestion
		TimeSeries: metrics.TimeSeries{Metric: j.foos, Tags: tags},
```

Reply:
Yeah, I am just making a 3rd commit to fix the xk6 tests, forgot about them when developing locally :sweat_smile:

Reply:
Resolved by https://github.com/grafana/k6/compare/7644b1c7dc4b05d84574ac315741dd43bc66cb2e..e1a641a1aa0c82d1d2f8db77611ccf4ba9bf301b